### PR TITLE
Setup Spork.

### PR DESCRIPTION
**Pull request by hermannloose**
This should make RSpec turnaround time much smaller, see updated `README`.

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -14,7 +14,8 @@ development:
 # Do not set this db to the same as development or production.
 test: &test
   adapter: sqlite3
-  database: ":memory:"
```
Comment (hermannloose) [inline config/database.yml:17]: See 91e81f9 as to why this was changed.

**Comment by hermannloose** [comment]
@jupp0r: Please review. Thanks.

**Comment by jupp0r** [comment]
Oops reviewed this a while ago and just forgot to comment on it. Do we already have enough specs to feel the impact of this, or is it more like a strategic decision to use spork? Ack anyway.

**Comment by hermannloose** [comment]
We do. In fact, the less specs you have, the more you see the improvement. We used to be pretty much Rails start-up time-bound, now Autotest starts running the actual specs right after hitting Save.

**Pull request merged**